cashu/wallet/api/router.py

```diff
@@ -127,6 +128,46 @@ async def invoice(
     return
 
 
+@router.post(
+    "/swap",
+    name="Multi-mint swaps",
+    summary="Swap funds between mints.",
+    response_model=SwapResponse,
+)
+async def swap(
+    amount: int = Query(default=..., description="Amount to swap between mints"),
+    outgoing_mint: str = Query(default=..., description="URL of outgoing mint"),
+    incoming_mint: str = Query(default=..., description="URL of incoming mint"),
+):
+    if not settings.lightning:
+        raise Exception("this mint does not support lightning.")
+    if outgoing_mint in incoming_mint:
```

Comment:
Why `in`? It'd be a bit odd but two distinct mints could be running at different URLs where one is a substring of the other, e.g. `https://example.com/cashu` and `https://example.com/cashu/staging`. Also weird but the same mint could also run at two different URLs (perhaps Tor/clearnet would be the most likely way to see this)

I *think* the only way to check if the mints are different is if they give a different `pubkey` in their `/info` response